lib/cdo/activity_constants.rb

```diff
@@ -10,5 +10,4 @@ module ActivityConstants
   MAXIMUM_NONOPTIMAL_RESULT = 29
   FREE_PLAY_RESULT = 30
   BEST_PASS_RESULT = 100
-  SUBMITTED_RESULT = 1000
```

Comment:
This makes this file diverge from /blockly/src/constants.js (see comment above)?